master/usage/removing-nodes.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Removing Nodes
+---
+
+With each Calico node there is a Node entry in the datastore and Workload
+Endpoints are associated with that node.  As nodes are replaced (with
+differently named nodes) or decomissioned, the datastore will accumulate
+node entries and possibly workload entries.  Currently there is not an
+automatic reaping of these old Node and Endpoint entries so it is necessary
+to periodically clean up these old entries or ensure they are removed when
+nodes are decomissioned.  This page provides instruction of how to remove
+those entries.
+
+> **Note**: Removing a Node entry will also remove the Workload Endpoint
+entries associated with that Node.
+
+### Removing a single Calico Node entry
+
+See the example below for how to remove a node with the calicoctl command.
+
+**Caution**
+- Before running the following make sure you are certain that the node is no
+  longer in service as all workloads associated with the node will be removed
+  from the datastore.
+- Any configuration specific to the node will be removed.  This would be
+  configuration like node BGP peerings or custom Felix configs.
+
+```
+calicoctl delete node <nodeName>
+```
+
+### Removing multiple Calico Node entries
+
+To remove several Nodes, a file can be created with several node entries and
+then be passed to the `calicoctl delete` command with the `-f` flag.
+Below is an example of steps to create a file of Nodes and delete them.
+
+1. Collect all node info
+   ```
+   calicoctl get nodes -o yaml > all_nodes.yaml
+   ```
+2. Create the file nodes_to_delete.yaml based off all_nodes.yaml with only
+   the nodes you want to remove.
+3. **Caution** Doing the step 5 with the all_nodes.yaml will most
```

Comment:
I know, I just thought it was a good way to include extra emphasis on the possibility of breaking things.  But I'll move them if you don't think that is a good way to do that.